fn_ansible_tower/fn_ansible_tower/lib/common.py

```diff
@@ -93,10 +96,21 @@ def project_pattern_match(results, project_pattern):
         # convert wildcard "*" to regex format ".*"
         pattern = project_pattern.replace("*", ".*")
         compiled = re.compile(pattern)
-        return [project for project in results if compiled.match(project['summary_fields']['project']['name'])]
```

Comment:
I get what you're doing. If's more pythonic to code like this:
return [project for project in results if project.get("summary_fields", {}).get("project", {}).get("name", None) and compiled.match(project['summary_fields']['project']['name'])]